src/main/java/io/getstream/chat/java/models/DeleteStrategy.java

```diff
@@ -0,0 +1,12 @@
+package io.getstream.chat.java.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum DeleteStrategy {
+  @JsonProperty("soft")
+  DEFAULT,
```

Comment:
I would suggest not providing this. Hard and soft are enough.

If missing is important then it means something semantically. Otherwise, good to leave it out (ignore and default).